### DISCUSSION TITLE
Extension installation failed :'(

### DISCUSSION THREAD

**Discussion by ClaraAmeyuri** [windingwind/zotero-better-notes#682] [Q&A] [answered]
Hi there, 

I followed the installation steps and recevied this **error message** : "**Extension installation failed**
The "%S" extension could not be installed. It may be incompatible with this version of Zotero."

I have the **last version of zotero** available on their website (**v.6**). 

_How can I resolve this problem_ ? 

Thanks in advance and have a nice day! <3

**Comment by windingwind** [comment, marked as answer]
https://github.com/windingwind/zotero-better-notes#-install

**Reply by ClaraAmeyuri** [reply]
Already followed the steps you mentionned. The error message appears at the end of the steps "Browse to where you downloaded the .xpi file and select it.". Also tried different version of better notes.

**Reply by windingwind** [reply]
[v1.0.4](https://github.com/windingwind/zotero-better-notes/releases/tag/1.0.4) (last for Zotero 6)
[v0.8.9](https://github.com/windingwind/zotero-better-notes/releases/tag/0.8.9) (last with auto-insert, tag-insert, math-ocr, for Zotero 6)

**Reply by ClaraAmeyuri** [reply]
Thank you so much for your quick response, it works. Your work is amazing. Gonna check how to support!
